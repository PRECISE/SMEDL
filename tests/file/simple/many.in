{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [1],
    "aux": {
        "arbitrary": "json data"
     }
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [2],
    "aux": [1, 2, 3]
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [3],
    "aux": "foo"
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [4],
    "aux": 36
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "in",
    "params": [5],
    "aux": {
    }
} 
