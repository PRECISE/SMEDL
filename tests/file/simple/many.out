{
    "fmt_version": [2, 0],
    "channel": "_simple_out",
    "event": "simple.out",
    "params": [1],
    "aux": {
        "arbitrary": "json data"
     }
} 
{
    "fmt_version": [2, 0],
    "channel": "_simple_out",
    "event": "simple.out",
    "params": [2],
    "aux": [1, 2, 3]
} 
{
    "fmt_version": [2, 0],
    "channel": "_simple_out",
    "event": "simple.out",
    "params": [3],
    "aux": "foo"
} 
{
    "fmt_version": [2, 0],
    "channel": "_simple_out",
    "event": "simple.out",
    "params": [4],
    "aux": 36
} 
{
    "fmt_version": [2, 0],
    "channel": "_simple_out",
    "event": "simple.out",
    "params": [5],
    "aux": {
    }
} 
